scripts/releaseChangelog.js

```diff
@@ -186,8 +193,8 @@ async function main(argv) {
   });
 
   const changelog = `
-## TODO RELEASE NAME
-<!-- generated comparing ${lastRelease}..${release} -->
+## __VERSION__
```

Comment:
It's easier to just double-click the placeholder, press <kbd>Ctrl</kbd> + <kbd>d</kbd> on VSCode and replace all versions at once.